src/github.ts

```diff
@@ -72,7 +72,7 @@ import {Update} from './updaters/update';
 // - make sure the branch name starts with "release"
 // - take everything else
 // This includes the tag to handle monorepos.
-const VERSION_FROM_BRANCH_RE = /^.*:release-(.*)+$/;
+const VERSION_FROM_BRANCH_RE = /^.*:release-?([\w-.]*)-(v[0-9].*)+$/;
```

Comment:
Why do we need this `+` in the end?  The `v[0-9].*` part will consume everything that starts from `v` followed by a digit, up to the end of the line.